source/validation-inline.php

```diff
@@ -111,6 +111,7 @@
     'prohibited'           => 'This field is prohibited.',
     'prohibited_if'        => 'This field is prohibited when :other is :value.',
     'prohibited_unless'    => 'This field is prohibited unless :other is in :values.',
+    'prohibits'            => 'This field field prohibits :other from being present.',
```

Comment:
*"This **field field** prohibits :other from being present."*,  **field** word repeated is right?

Reply:
of course not, thanks for the review.